Assets/MRTK/Providers/LeapMotion/LeapMotionArticulatedHand.cs

```diff
@@ -246,7 +244,7 @@ protected void UpdateInteractions()
             MixedRealityPose indexPose = jointPoses[TrackedHandJoint.IndexTip];
 
             // Only update the hand ray if the hand is in pointing pose
-            if (handDefinition.IsInPointingPose)
+            if (IsInPointingPose)
```

Comment:
Interesting, how did this work before?

Reply:
This was a simplification,  I realized that the `handDefinition.IsInPointingPose` could be simplified to just `IsInPointingPose` since I have ` IsInPointingPose => handDefinition.IsInPointingPose;` at the top of the file.